Dockerfiles/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2017 Intel Corporation
```

Comment:
should this file be renamed to `Dockerfile.buildtest` or something similar? 
I suspect we will need to add more Dockerfiles in this directory.